src/dataEditor/dataEditorClient.ts

```diff
@@ -60,34 +59,35 @@ import {
   ViewportEvent,
   ViewportEventKind,
 } from '@omega-edit/client'
+import assert from 'assert'
+import fs from 'fs'
+import net from 'net'
+import os from 'os'
 import path from 'path'
+import * as vscode from 'vscode'
 import XDGAppPaths from 'xdg-app-paths'
-import assert from 'assert'
-import { SvelteWebviewInitializer } from './svelteWebviewInitializer'
+import { extractDaffodilEvent } from '../daffodilDebugger/daffodil'
+import {
+  EditByteModes,
+  VIEWPORT_CAPACITY_MAX,
+} from '../svelte/src/stores/configuration'
```

Comment:
I'm confused as to why these are marked as additions. Especially the removal of `import fs from 'fs'` on line 18 and then it's added again on line 63. Same goes for `net`, `os`, and `vscode`...

I feel as if there's been a rebasing issue here.

Reply:
Not a rebasing issue, just VSCode sorting the import statements on a save.